src/server/api/lib/twilio.js

```diff
@@ -109,15 +109,26 @@ async function sendMessage(message) {
     if (message.service !== 'twilio') {
       log.warn('Message not marked as a twilio message', message.id)
     }
-    const messageParams = {
+
+    // Image extraction
+    const extractor = new RegExp(/\[.*http(.*)\]/)
```

Comment:
I think the first `.*` should be `\s*` and the second you probably want `([^\]]*)` to be a little safer